Mobile Buy SDK/Mobile Buy SDK Tests/BUYRequestOperationTests.m

```diff
@@ -332,7 +332,7 @@ - (void)testCancellationDuringPolling
 	
 	[self.queue addOperation:operation];
 	
-	[self createExpectationDelay:1.0 block:YES];
+	[self createExpectationDelay:10.0 block:YES];
```

Comment:
I know.  We could be a little more aggressive, say 3.0 seconds, and back it off if we get some failures.  The 1.0s timeout was demonstrating some failures due to a race condition.